maxima.yaml

```diff
@@ -0,0 +1,39 @@
+name: maxima
+sources:
+  - type: archive
+    url: https://downloads.sourceforge.net/maxima/maxima-5.42.1.tar.gz
+    sha256: 8f555aec33bc61b5a3ee0fe2e9d6c1179db67a2ff7e0eceb6bb614058eeb40cd
+  # https://bugzilla.redhat.com/show_bug.cgi?id=837142
+  # https://sourceforge.net/tracker/?func=detail&aid=3539587&group_id=4933&atid=104933
+  # https://src.fedoraproject.org/rpms/maxima/raw/66de1aea9ace767c984ad32091ee25409e2a70be/f/maxima-5.37.1-clisp-noreadline.patch
+  - type: patch
+    path: maxima-5.37.1-clisp-noreadline.patch
+  # Build the fasl while building the executable to avoid double initialization
+  # https://src.fedoraproject.org/rpms/maxima/raw/66de1aea9ace767c984ad32091ee25409e2a70be/f/maxima-5.30.0-build-fasl.patch
```

Comment:
Has it been upstreamed?

Reply:
@barthalion 

> Has it been upstreamed?

No, and to be honest, I don't care. These patches are for `maxima`, not for `wxMaxima` itself. Moreover, they are used by many distributions, including Fedora, Gentoo, Arch, probably more. I don't know why these patches have not yet been upstreamed and I don't really care. The maintainers of Linux distributions should take care of this, because they know the exact meaning of these fixes.

Reply:
You also are a maintainer if we accept this.

Reply:
It also doesn't sound reassuring if you are blindly applying patches without knowing what they are for.

Reply:
If I remember correctly, there was a discussion on the mailing list, but it was cut off at some point. The idea was that it should be integrated into the Makefile, but just no one took it upon themselves to do that.

Reply:
I couldn't get it to build without these patches. I honestly don't care what they are, as long as they fulfill their function.